src/schema/schema.js

```diff
@@ -63,7 +65,10 @@ export function setNodeAttrs(node, extraAttrs) {
 //       A function to add the attribute's value to an attribute
 //       object that's used to render the cell's DOM.
 export function tableNodes(options) {
-  const tableCellExtraAttrs = {...options.cellAttributes || {}, ...cellExtraAttrs} ;
+  const tableCellExtraAttrs = {
```

Comment:
seems like all linting changes? can we in a future PR just use the same linting setup as `editor-mvp`?